recipes/scevan/meta.yaml

```
@@ -0,0 +1,56 @@
+package:
+  name: scevan
```

I think that the package name should have `r-` appended.
```suggestion
  name: r-scevan
```